minimum-constraints.txt

```diff
@@ -177,7 +177,7 @@ tox==2.5.0
 
 # Sphinx (no imports, invoked via sphinx-build script):
 Sphinx==1.7.6; python_version <= '3.4'
-Sphinx>= 3.5.4; python_version >= '3.5'
+Sphinx==3.5.4; python_version >= '3.5'
```

Comment:
I suggest adding the same minimum version for more-itertools as in dev-requirements.txt:
```
more-itertools==4.0.0
```

Reply:
DONE